xs2a-adapter-aspsp-registry/src/main/resources/aspsp-adapter-config.csv

```diff
@@ -1 +1,5 @@
 53c47f54-b9a4-465a-8f77-bc6cd5f0cf46,adorsys xs2a,ADORSYS,https://amos-modelbank-xs2a.cloud.adorsys.de,adorsys-integ-adapter,99999999
+2dfd45da-64cd-4023-94a5-671f664b5eec,Bank Of Alpha,ALPHA,https://amos-modelbank-xs2a.cloud.adorsys.de,adorsys-integ-adapter,11111111
```

Comment:
Column names are UUID, bank name, BIC, API URL, adapter ID, bank code(BLZ), IDP URL.
If you wanna use external sandboxes then API URL and bank code should be taken from their developer portal.
In case when you want to use the same sandbox but a different bank name then you should create a new bank account with bank code 11111111, 22222222, et. Because the bank code is using as a reference to the specific bank.

Reply:
So this could be accepted?
We don't need external sandboxes at the moment.